README.md

```diff
@@ -59,7 +59,7 @@ $ shfmt -l -w nrk-tv-downloader.sh
 ```
 
 ## Requirements
-This script requires bash, rev, cut, grep, sed, gawk, printf, curl and ffmpeg/avconv.
+This script requires bash, rev, cut, grep, sed, gawk, printf, curl and ffmpeg/avconv, [jq](https://stedolan.github.io/jq/).
```

Comment:
Hey! Would be nice to keep the grammar correct and keep `and` before the last element in the list.